style.css

```diff
@@ -0,0 +1,301 @@
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    width: 96vw;
+    margin: auto;
+}
+
+hr {
+    margin-bottom: 0.3rem;
+    background-color: #eee;
+    border: 0 none;
+    color: #eee;
+    height: 1px;
+}
+
+h1 {
+    font-size: 3.8rem;
+    font-weight: 700;
+    text-align: center;
+}
+
+h2 {
+    font-size: 1.6rem;
+    font-weight: 1000;
+    font-style: italic;
+    padding: 0rem 2rem;
+    line-height: 1.8rem;
+    text-align: center;
+    margin: 3rem 0rem 1rem 0rem;
+    background-color: rgba(255, 204, 53, .4);
+    transform: skew(9deg, 0deg);
+}
+
+h3 {
+    font-size: 0.8rem;
+    margin: 0.75rem 0rem 1.5rem 0rem;
+    color: #ED1C24;
+}
+
+a {
+    color: black;
+    text-decoration: none;
+}
+
+header {
+    box-shadow: 0 4px 4px -2px rgba(0, 0, 0, 0.15);
+}
+
+button:hover {
+    cursor: pointer;
+}
+
+/* Upper header styles */
+.header-wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 1.4rem 2rem 0.5rem 2rem;
+}
+
+.header-left {
+    display: flex;
+    align-items: center;
+    flex: 5 1 auto;
+    font-size: 0.65rem;
+    font-weight: 600;
+}
+
+.header-left img {
+    margin-right: 1.5rem;
+    height: 12px;
+}
+
+.nav-wrapper {
+    display: flex;
+    flex: 1 1 auto;
+}
+
+.nav-links {
+    display: flex;
+    flex: 1 1 auto;
+    justify-content: space-between;
+    list-style: none;
+}
+
+.nav-link {
+    font-size: 0.65rem;
+    font-weight: 600;
+    flex-wrap: wrap;
+    display: flex;
+    justify-content: space-evenly;
+    padding-left: 0.5rem;
+}
+
+/* Lower header Styles */
+.header-lower-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 0rem 2rem 0.3rem 2rem;
+}
+
+.header-lower-wrapper button {
+    font-family: "Poppins", sans-serif;
+    font-weight: 600;
+}
+
+.header-logo {
+    display: flex;
+    align-items: center;
+}
+
+.header-logo img {
+    width: 4.5rem;
+    padding-right: 1rem;
+}
+
+.search-bar-wrapper {
+    display: flex;
+    height: 2rem;
+    justify-content: space-between;
+    align-items: center;
+    flex: 2 1 30vw;
+    background: #eee;
+    border: 1px solid transparent;
+    position: relative;
+}
+
+.search-wrapper-2 {
+    display: flex;
+}
+
+.magnifying-glass-wrapper {
+    display: flex;
+    align-items: center;
+    background: #eee;
+    height: 2rem;
+}
+
+.magnifying-glass {
+    height: 15px;
+    padding: 0px 10px;
+}
+
+#search {
+    font-family: "Poppins", sans-serif;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    width: 100%;
+    height: 100%;
+    border: none;
+    outline: none;
+    background: #eee;
+    color: rgba(0, 0, 0, 0.5);
+}
+
+.search-button {
+    padding: 0.5rem 1rem;
+    border-style: none;
+    background: #fb3e44;
```

Comment:
Perhaps consider using consistently RGB/HEX/"human language" colours, instead of a mix of it.